guest/testdata/cyclestate/main.go

```diff
@@ -178,7 +180,6 @@ func (statePlugin) PreBind(state api.CycleState, pod proto.Pod, _ string) (statu
 	if unsafe.Pointer(pod.Spec()) != unsafe.Pointer(podSpec) {
 		panic("didn't cache pod from score")
 	}
-	mustFilterState(state)
```

Comment:
Any reason you removed it?

Reply:
This is for `Test_guestPool_bindingCycles` test case.
In this test, postBind was called but preBind, Bind and Filter were not called.
In order to test postBind's cycleState, I changed to call preBind and Bind in this test.
But in this test, Filter is not called. So I removed this mustFilterState statement.